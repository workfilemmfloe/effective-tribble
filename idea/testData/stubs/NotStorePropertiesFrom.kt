class Test() {
    val test = 12;
    {
        for (i in 0..12) {
        }
    }
    fun more() {
    }
}
