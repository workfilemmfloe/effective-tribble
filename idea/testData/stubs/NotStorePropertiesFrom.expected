PsiJetFileStubImpl[package=]
  PACKAGE_DIRECTIVE:
  CLASS:[fqName=Test, isEnumEntry=false, isLocal=false, isTopLevel=true, isTrait=false, name=Test, superNames=[]]
    VALUE_PARAMETER_LIST:
    CLASS_BODY:
      PROPERTY:[fqName=Test.test, hasDelegate=false, hasDelegateExpression=false, hasInitializer=true, hasReceiverTypeRef=false, hasReturnTypeRef=false, isTopLevel=false, isVar=false, name=test]
      ANONYMOUS_INITIALIZER:
      FUN:[fqName=Test.more, hasBlockBody=true, hasBody=true, hasTypeParameterListBeforeFunctionName=false, isExtension=false, isTopLevel=false, name=more]
        VALUE_PARAMETER_LIST:
